pyface/ui/qt4/init.py

```diff
@@ -18,17 +18,6 @@
 from pyface.base_toolkit import Toolkit
 from .gui import GUI
 
-if qt_api == "pyqt":
```

Comment:
I'll check.  There are a few things done in the `init.py` to make sure that we have a valid environment and Qt has been initialized to a working state.  It is possible `QtCore` is imported just to prove it can be.

Reply:
Nope - it's redundant.  Will remove.